swap/src/protocol/alice/event_loop.rs

```diff
@@ -172,6 +177,13 @@ where
                         }
                         SwarmEvent::ConnectionEstablished { peer_id: peer, endpoint, .. } => {
                             tracing::debug!(%peer, address = %endpoint.get_remote_address(), "New connection established");
+
+                            if let Some(transfer_proof) = self.buffered_transfer_proofs.remove(&peer) {
+                                tracing::debug!(%peer, "Found buffered transfer proof for peer");
+
+                                let result = self.swarm.send_transfer_proof(peer, transfer_proof);
+                                assert!(result.is_ok(), "must be able to send transfer proof after connection was established");
```

Comment:
❓ What is the reason that you use an `assert` here instead of using `expect` directly on the send call ?

Reply:
No reason actually, just didn't think of `expect` :)